Mihai-Blaga/COMP30022
# Tasks/AA-79-Login-Page-UI

Changes:
- Added new logo and sign in with Google button

To test:
- Clear storage for app (hold click the CRM app icon, go into app info etc.)
- Open up app and screen should load
